### PR TITLE
migrate ember projects to ce3

### LINKED ISSUE #4085: Port ember-client to cats-effect-3

### LINKED ISSUE #4086: Port ember-server to cats-effect-3

### PR DESCRIPTION
**Pull request by m-sp**
instead of https://github.com/http4s/http4s/pull/4125 and https://github.com/http4s/http4s/pull/4128

fixes #4086
fixes #4085

### CODE REVIEW

File: ember-client/src/test/scala/org/http4s/ember/client/internal/ClientHelpersSpec.scala [hunk 1]
```diff
@@ -159,31 +157,31 @@ class ClientHelpersSpec extends Specification with CatsIO {
...
+//              reuse
+//            )
+//            .use { resp =>
+//              resp.body.compile.drain.attempt >>
+//                reuse.get.map { case r =>
+//                  r must beEqualTo(Reusable.DontReuse)
+//                }
+//            }
+//      } yield testResult
+//    }.pendingUntilFixed
```
Comment (m-sp) [inline ember-client/src/test/scala/org/http4s/ember/client/internal/ClientHelpersSpec.scala:184]: any test expert knows how to fix that?

**Review by yanns — APPROVED** [review]
🥇

File: ember-client/src/test/scala/org/http4s/ember/client/internal/ClientHelpersSpec.scala (hunk 1 above)
Comment (rossabaker) [reply ember-client/src/test/scala/org/http4s/ember/client/internal/ClientHelpersSpec.scala:183]: Do we have an issue on this so it doesn't get forgotten, at least?

**Review by rossabaker — APPROVED** [review]

File: ember-client/src/test/scala/org/http4s/ember/client/internal/ClientHelpersSpec.scala (hunk 1 above)
Comment (m-sp) [reply ember-client/src/test/scala/org/http4s/ember/client/internal/ClientHelpersSpec.scala:183]: here you go https://github.com/http4s/http4s/issues/4257

**Pull request merged**